### PR TITLE
[workers] add wrangler v3 dependency info for updating

### PR DESCRIPTION
**Pull request by admah**
Added note about better-sqlite3 dependency for Wrangler v3 per #9463 

For more information: https://github.com/cloudflare/workers-sdk/issues/3531

### CODE REVIEW

**Review by WalshyDev — APPROVED** [review]

**Comment by deadlypants1973** [comment]
Does this solve https://github.com/cloudflare/cloudflare-docs/issues/10230 ?

File: content/workers/wrangler/migration/update-v2-to-v3.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ If you tried to update to Wrangler v3 prior to v3.3, you may have experienced so
 
 {{</Aside>}}
 
+## Additional dependencies
+
+As a result of updating the Miniflare storage system, `better-sqlite3` is now a dependency of Wrangler v3. If you do not have this installed, view the package's [installation instructions](https://www.npmjs.com/package/better-sqlite3) on npm for more information.
```
Comment (deadlypants1973) [inline content/workers/wrangler/migration/update-v2-to-v3.md:19]:
```suggestion
As a result of updating the Miniflare storage system, `better-sqlite3` is now a dependency of Wrangler v3. If you do not have `better-sqlite3` installed, view the [`better-sqlite3` installation instructions](https://www.npmjs.com/package/better-sqlite3) on npm for more information.
```

**Review by deadlypants1973 — APPROVED** [review]
PCX

**Comment by lrapoport-cf** [comment]
following https://github.com/cloudflare/miniflare/pull/656, this docs update is no longer needed 👍

**Pull request closed without merging**